src/ui/sign_message_nbgl.c

```diff
@@ -0,0 +1,145 @@
+#ifdef HAVE_NBGL
+
+#include "io.h"
+#include "menu.h"
+#include "sol/parser.h"
+#include "sol/printer.h"
+#include "sol/print_config.h"
+#include "sol/message.h"
+#include "sol/transaction_summary.h"
+#include "glyphs.h"
+#include "apdu.h"
+#include "utils.h"
+
+#include "nbgl_page.h"
```

Comment:
```suggestion
```
Not needed as you only use use_cases.

Reply:
Well I define a `nbgl_pageInfoLongPress_t` static struct